index.d.ts

```diff
@@ -18,4 +18,23 @@ for await (const chunk of stream) {
 }
 ```
 */
-export default function mergeStreams(streams: NodeJS.ReadableStream[]): NodeJS.ReadableStream;
+export default function mergeStreams(streams: NodeJS.ReadableStream[]): MergedStreams;
+
+/**
+Single stream combining the output of multiple streams.
+*/
+export type MergedStreams = NodeJS.ReadableStream & {
```

Comment:
```suggestion
export type MergedStream = NodeJS.ReadableStream & {
```

It returns a single stream. Having it plural makes it sound like it returns a list of streams.

Reply:
Done in a6f9ab3a656b6950b5ec71afd3ef7621d6e1871b.